Add fixed NFT supply topic (#1750)
XRPLF/xrpl-dev-portal: content/concepts/tokens/nft-fixed-supply.md

---
html: nft-fixed-supply.html
parent: non-fungible-tokens.html
blurb: Introduction to XRPL NFTs.
labels:
  - Non-fungible Tokens, NFTs
---

# Guaranteeing a Fixed Supply of NFTs

> For some projects, you might want to guarantee that no more than a fixed number of NFTs are minted from a given issuing account. Use the following steps to achieve that goal.

Reviewer: ```suggestion
For some projects, you might want to guarantee that no more than a fixed number of NFTs are minted from an issuing account.
```

To guarantee a fixed number of NFTs:

1. Create and fund a new account, the _Issuer_. This account is the issuer of the tokens within the collection. See [Creating Accounts](accounts.html#creating-accounts).
1. Use `AccountSet` to assign your operational wallet as an authorized minter for the issuer. See [Authorizing Another Account to Mint Your NFTs](nftoken-authorized-minting.html).

> 1. Use your operational account to mint the tokens using NFTokenMint. The operational wallet holds all of the tokens minted for the Issuer. See [Batch Minting](nftoken-batch-minting.html).

Reviewer: ```suggestion
1. Use your operational account to mint the tokens using `NFTokenMint`. The operational wallet holds all of the tokens minted for the Issuer. See [Batch Minting](nftoken-batch-minting.html).
```
